internal/provider/resource_gitlab_topic_test.go

```diff
@@ -256,6 +258,46 @@ func TestAccGitlabTopic_softDestroy(t *testing.T) {
 	})
 }
 
+func TestAccGitlabTopic_titleSupport(t *testing.T) {
+	rInt := acctest.RandInt()
+
+	resource.Test(t, resource.TestCase{
+		ProviderFactories: providerFactories,
+		CheckDestroy:      testAccCheckGitlabTopicDestroy,
+		Steps: []resource.TestStep{
+			{
+				SkipFunc: isGitLabVersionAtLeast(context.Background(), testGitlabClient, "15.0"),
+				Config: fmt.Sprintf(`
+					resource "gitlab_topic" "this" {
+						name = "foo-%d"
+						title = "Foo-%d"
+					}
+				`, rInt, rInt),
+				ExpectError: regexp.MustCompile(`title is not supported by your version of GitLab. At least GitLab 15.0 is required`),
```

Comment:
[regexp.QuoteMeta](https://pkg.go.dev/regexp#QuoteMeta) is better for creating a regex that matches a string literal

Reply:
`regexp.QuoteMeta` returns a string, not a regex. Since this code is copied, I think I'll leave it as-is. Are you okay with that?

Reply:
Actually, I cherry-picked it from my own code from https://github.com/gitlabhq/terraform-provider-gitlab/pull/1048. Sorry 🙈 

Still, I think it's easier to use `MustCompile` here, or should I first do the `QuoteMeta` and then turn into a regex?

Reply:
In your case I don't think it's a big deal. The `.` characters technically mean "any character" in your regex, but realistically it doesn't matter. I was thinking back to other test code, where the error message sometimes has wackier characters, as in: https://github.com/gitlabhq/terraform-provider-gitlab/blob/89b7cd6e969e4efe4e748859bf48edfe980d9d96/internal/provider/resource_gitlab_project_variable_test.go#L168-L170

Reply:
Yes, given that it's "just test code" I was fine with it :)